PR_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Description
```

Comment:
I figured I'd add this while I was in here; I've found them to be helpful. Since this isn't my project, happy to back it out if it's not wanted/needed :)